### PR TITLE
Add Acecad Flair / Pentagram Quadpen to hwdb

### PR DESCRIPTION
**Pull request by cprn**
Acecad Flair tablet is missing `resolution` capability. This entry makes it work.

Other names / brandings of this tablet include:
- (US) Acecat Flair
- (Western Europe) Acecad Flair
- (Central Europe) Pentagram Quadpen
- (`lsusb`) ID 0460:0004 Ace Cad Enterprise Co., Ltd Tablet (5x3.75)
- (`xinput list`) ACECAD USB Graphics Tablet  Pen (0)

### CODE REVIEW

**Comment by cprn** [comment]
Resolution values were selected on a hunch but they work pretty well. If you can suggest better values based on the tablet's sensor resolution I'll be happy to update this MR. From the original driver's source:
```c
input_set_abs_params(input_dev, ABS_X, 0, 5000, 4, 0);
input_set_abs_params(input_dev, ABS_Y, 0, 3750, 4, 0);
```
Active area size: 5 x 3.75" (127 x 95 mm).

**Comment by bluca** [comment]
> Resolution values were selected on a hunch but they work pretty well. If you can suggest better values based on the tablet's specification I'll be happy to update this MR - personally I have no idea how to calculate those. From the original driver's source:
> 
> ```c
> input_set_abs_params(input_dev, ABS_X, 0, 5000, 4, 0);
> input_set_abs_params(input_dev, ABS_Y, 0, 3750, 4, 0);
> ```
> 
> active area: 127 x 95.25 mm (5 x 3.75")
> 2540 lines per inch
> 1024 pressure levels
> 0.254 mm precision (0.01")

@whot @hadess any suggestion on the above?

**Comment by hadess** [comment]
> @whot @hadess any suggestion on the above?

I don't work in this area, so I have no idea, sorry.

**Comment by whot** [comment]
The resolution is in units/mm, so the simple formula is: `kernel range / physical size = resolution`. So in your case, for X it looks like: 5000/127 = 39 or 40, depending which side of the rounding error you want to be on.

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -38,6 +38,15 @@
...
 # Sort by brand, model
 
+#########################################
+# ACECAD
+#########################################
+
+# Acecad Flair / Pentagram Quadpen
+evdev:input:b0003v0460p0004*
+ EVDEV_ABS_00=::20
+ EVDEV_ABS_01=::20
```
Comment (keszybz) [inline hwdb.d/60-evdev.hwdb]: So this should be `::40` IIUC.

Comment (whot) [reply hwdb.d/60-evdev.hwdb]: correct, apologies, should've said so explicitly

Comment (cprn) [reply hwdb.d/60-evdev.hwdb]: I confirm these settings work even better ("snappier" is the best word I can think of to describe it). I pushed that commit and rebased to sync with upstream after (i.e. with this repository) - sorry if that looks bad in the history. Will look into doing it cleaner in the future.

**Review by bluca — APPROVED** [review]
lgtm

**Pull request merged**